e2e/tests/workspace/workspace-edit.spec.ts

```diff
@@ -0,0 +1,73 @@
+import DataPage, {TabLabelAlias} from 'app/page/data-page';
+import WorkspacesPage from 'app/page/workspaces-page';
+import {EllipsisMenuAction} from 'app/text-labels';
+import * as testData from 'resources/data/workspace-data';
+import {findWorkspace, performActions, signIn} from 'utils/test-utils';
+import WorkspaceAboutPage from 'app/page/workspace-about-page';
+
+
+describe('Editing workspace thru workspace card ellipsis menu', () => {
+
+  beforeEach(async () => {
+    await signIn(page);
+  });
+
+  /**
+   * Test:
+   * - Find an existing workspace. Create a new workspace if none exists.
+   * - Edit workspace: changing Question 2 answers.
+   * - Verify landing in DATA tab.
+   * - Verify Workspace Information in ABOUT tab.
+   */
+  test('User as OWNER can edit workspace', async () => {
+    const workspaceCard = await findWorkspace(page);
+    await (workspaceCard.getEllipsis()).clickAction(EllipsisMenuAction.Edit);
+
+    const workspacesPage = new WorkspacesPage(page);
+    // Change question #2 answer
+    await performActions(page, testData.defaultAnswersResearchPurposeSummary);
```

Comment:
how does this work?

Reply:
Basically, iterate thru a json object consisted of elements and performs action (type/check/select) according to the type (textinput/checkbox/radiobutton) of element. (saves me from dupl. code)

```
export const defaultAnswersResearchPurposeSummary = [
  {
    id: FIELD.RESEARCH_PURPOSE_SUMMARY.scientificQuestionsIntentToStudyTextarea,
    value: makeString(100)
  }, {
    id: FIELD.RESEARCH_PURPOSE_SUMMARY.scientificApproachesToUseTextarea,
    value: makeString(100)
  }, {
    id: FIELD.RESEARCH_PURPOSE_SUMMARY.anticipatedFindingsFromStudyTextarea,
    value: makeString(100)
  }
];
```

Reply:
oh ok - it fills in those 3 text fields.  cool.